### PR TITLE
chore: add FeedSingleTweet using previous layout of tweet in FeedPost

### PR DESCRIPTION
**Pull request by Mykyta-Pysarev**
Add feed single tweet (default tweet layout in the feed). I reused and change original layout from FeedPost.
image

### CODE REVIEW

**Comment by Mykyta-Pysarev** [comment]
Fix merge conflict

**Review by ArtemLigerko — APPROVED** [review]

**Review by VladislavPokhyl — APPROVED** [review]

File: src/components/FeedSingleTweet.tsx [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                    <Popover.Body>
+                      <p>
+                        <StyledLink
+                          className="text-decoration-none text-reset d-flex flex-row p-1 pe-2 fs-6"
+                          href="#"
+                        >
+                          <span className="px-2">
+                            <PersonX />
+                          </span>
+                          Не читати {userNickname}
```
Comment (VitaliiDenysiuk) [inline src/components/FeedSingleTweet.tsx]: use i18n

File: src/components/FeedSingleTweet.tsx [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                      </p>
+                      <p>
+                        <StyledLink
+                          className="text-decoration-none text-reset d-flex flex-row p-1 pe-2 fs-6"
+                          href="#"
+                        >
+                          <span className="px-2">
+                            <ClipboardPlus />
+                          </span>
+                          Додати {userNickname} до списку
```
Comment (VitaliiDenysiuk) [inline src/components/FeedSingleTweet.tsx]: same

File: src/components/FeedSingleTweet.tsx [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+                      </p>
+                      <p>
+                        <StyledLink
+                          className="text-decoration-none text-reset d-flex flex-row p-1 pe-2 fs-6"
+                          href="#"
+                        >
+                          <span className="px-2">
+                            <Flag />
+                          </span>
+                          Поскаржитись на твіт
```
Comment (VitaliiDenysiuk) [inline src/components/FeedSingleTweet.tsx]: same

**Review by KirillSerg — APPROVED** [review]
great job

File: src/components/FeedSingleTweet.tsx [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+                trigger="click"
+                key="left"
+                placement="left"
+                overlay={
+                  <StyledPopover id="popover-positioned-left">
+                    <Popover.Body>
+                      <p>
+                        <StyledLink
+                          className="text-decoration-none text-reset d-flex flex-row p-1 pe-2 fs-6"
+                          href="#"
```
Comment (vasyagladush) [inline src/components/FeedSingleTweet.tsx:149]: Do we need href="#" ?

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**